src/qute/commands/registerCommands.ts

```diff
@@ -31,6 +31,7 @@ export function registerVSCodeQuteCommands(context: ExtensionContext) {
 }
 
 export function registerQuteExecuteWorkspaceCommand(context: ExtensionContext, languageClient: LanguageClient) {
```

Comment:
Since this registers more commands than just `QuteClientCommandConstants.EXECUTE_WORKSPACE_COMMAND` now, I think you should rename it to something like `registerQuteLSDependentCommands`